Android.mk

```diff
@@ -21,61 +21,61 @@ rwildcard=$(wildcard $1$2) $(foreach d,$(wildcard $1*),$(call rwildcard,$d/,$2))
 # Creating prebuilt for dependency: modloader - version: 1.2.3
```

Comment:
Android mk and application mk are not needed anymore for qpm rust, these should be removed from the repo

(git rm ./Android.mk && git rm ./Application.mk)